src/_shared/js/capi-images.js

```diff
@@ -0,0 +1,65 @@
+// Creates a set of source elements for an image.
+function buildSources (sourceData) {
+
+	let sourcesFragment = document.createDocumentFragment();
+
+	return sourceData.reduce((sources, source) => {
```

Comment:
Would it make sense to use forEach here instead of reduce because of side-effects? @regiskuckaertz